assets/js/modules/ads/components/settings/SettingsForm.stories.js

```diff
@@ -29,12 +29,10 @@ import { Cell, Grid, Row } from '../../../../material-components';
 import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
 import { MODULES_ADS } from '../../datastore/constants';
 import { MODULE_SLUG_ADS } from '../../constants';
-import {
-	provideModules,
-	WithTestRegistry,
-} from '../../../../../../tests/js/utils';
+import { provideModules } from '../../../../../../tests/js/utils';
+import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
 
-function Template( args ) {
+function Template( { args } ) {
```

Comment:
This `args` should not be destructured unless it's referring to a specific prop.